Dockerfile

```diff
@@ -60,10 +60,10 @@ ENV PATH="/opt/conda/bin:$PATH"
 RUN conda install -qy conda==23.1.0 \
     && conda install -y -c conda-forge \
       openmm=7.5.1 \
-      cudatoolkit==${CUDA_VERSION} \
       pdbfixer \
       pip \
-      python=3.8 \
+      python=3.9 \
+      && conda install -y -c nvidia/label/cuda-11.8.0 cuda-libraries-dev cuda-nvcc cuda-nvtx cuda-cupti \
```

Comment:
Hi @cjpooley , just curious, in this line, why we don't have cuda-toolkit which would be equivalent to previous version ? Instead it has a series of multiple cuda components?

Reply:
There was a change made to the cudatoolkit so that at least by version 11.8 it no longer included required components, including nvcc. This caused errors in my initial testing, so I followed the guidance [here](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#installing-cuda-using-conda:~:text=If%20you%20had%20previously%20installed%20CUDA%20using%20the%20cudatoolkit%20package%20and%20want%20to%20maintain%20a%20similar%20install%20footprint%2C%20you%20can%20limit%20your%20installation%20to%20the%20following%20packages%3A).

Thanks for the suggestion on referencing the CUDA arg - will do.

Reply:
Not sure if you have A100 quota handy to replicate it.